Mention TS example repo on the website (#1786)
nextauthjs/next-auth: www/docs/getting-started/typescript.md

NextAuth.js comes with its own type definitions, so you can safely use it in your TypeScript projects. Even if you don't use TypeScript, IDEs like VSCode will pick this up, to provide you with a better developer experience. While you are typing, you will get suggestions about how certain objects/functions look like, and sometimes also links to documentation, examples and other useful resources.

> We made an example repository showcasing how to use `next-auth` on a NextJS application with Typescript:  

Reviewer: ```suggestion
Check out the example repository showcasing how to use `next-auth` on a Next.js application with TypeScript:  
```
